src/components/phase/PhaseApp.js

```diff
@@ -60,6 +65,7 @@ class PhaseApp extends Component {
     appInstance: null,
     name: 'Image',
     lang: DEFAULT_LANGUAGE,
+    geolocation: undefined,
```

Comment:
Not good practice to set default to undefined, because that will trigger the warning from the React props parser. Set to `null` instead.